modules/rtm/R/invert.custom.R

```diff
@@ -242,32 +262,41 @@ invert.custom <- function(observed, invert.options,
     # Metropolis sampling step if all conditions have been met
```

Comment:
how are the checks on [L.237](https://github.com/ashiklom/pecan/blob/40f0fe130286dd91898bf5cc012f668b1a321feb/modules/rtm/R/invert.custom.R#L237) and [L.257](https://github.com/ashiklom/pecan/blob/40f0fe130286dd91898bf5cc012f668b1a321feb/modules/rtm/R/invert.custom.R#L257) are different? I mean, why would a value within `param.mins` and `param.mins` fall outside the prior? 

Also, I was just wondering, instead of skipping to the next iteration right away, how it would affect to re-propose until all your parameters are valid and only then you calculate the metropolis ratio?